src/Views/Components/AdminUI/api.ts

```diff
@@ -32,4 +32,39 @@ const usePostRequest = (endpoint: string, apiNonce: string, successMessage: stri
 
     return {postData, result};
 };
-export {usePostRequest};
+
+const useGetRequest = (endpoint: string, apiNonce: string, successMessage: string, errorMessage: string) => {
+    const [result, setResult] = useState({type: null, message: ''});
```

Comment:
We are currently passing empty strings for the success & error messages. Im not sure we plan on using them to display anything on the frontend but I may be wrong. If not we probably can just remove it.

Reply:
Resolved 91a6e400f00ab56d0fd13a3441d7e566d8dac5e9